txnprovider/txpool/pool_txn_parser.go

```diff
@@ -346,7 +346,7 @@ func (ctx *TxnParseContext) parseTransactionBody(payload []byte, pos, p0 int, sl
 	}
 
 	if slot.Type == AATxnType {
-		return parseTransactionBodyAA(payload, p, slot)
+		return parseTransactionBodyAA(payload, p, slot, sender)
```

Comment:
@shohamc1 
I was getting "insufficient funds" errors and it seemed to be linked with txpool checking this value for the `sender` address. This needs to be modified with Paymaster support, I assume.